### PR TITLE
Add pages referring to reference application

### PR DESCRIPTION
**Pull request by hswopeams**
Draft PR for adding reference application info

### CODE REVIEW

File: src/docs/reference-application.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: docs
+title: "Boson Protocol: Docs: Reference Application"
+short_title: "Reference Application"
+permalink: /reference-application/
+---
+
+# Leptonit application
```
Comment (atkinsonholly) [inline src/docs/reference-application.md:8]: Typo Leptonite

File: src/docs/reference-application/development-setup.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+In order to contribute to the Leptonite application or further develop your own
+forked copy of the Leptonite application, you will need to set up a local
+development environemnt. This page explains how to do that. These instructions
+assume a working knowledge of git and GitHub.
+
+## Repositories
+
+By default, v0.1.0 of the reference application points to v0.1.0 of the Boson
+Protocol smart contracts running on the Rinkeby testnet.
```
Comment (atkinsonholly) [inline src/docs/reference-application/development-setup.md:18]: deployed to the Rinkeby testnet.

File: src/docs/reference-application/introduction.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+# Introduction to the Leptonite application
+
+The Leptonite applications shows you how to build a p2p marketplace powered by
+Boson Protocol. Users can connect their wallets and list a set of items as a
+seller, as well as discover products that can be purchased as a buyer. The
+application also demonstrates how the transaction lifecycle can be tracked and
+co-ordinated by both parties.
+
+1. **NFTs for Physical** - Sellers can create listings for physical items that
```
Comment (atkinsonholly) [inline src/docs/reference-application/introduction.md:16]: Change 'NFTs for Physical' to 'Digital to Physical'

File: src/docs/reference-application/introduction.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+2. **Minting NFT on commitment** - Buyers can purchase items, which mints an NFT
+   on commitment and payment
+3. **Automated escrow** - A commitment NFT's lifecycle can be tracked by both
+   buyer and seller with each being incentivised to complete a successful
+   redemption with payment being transferred and deposits being returned.
+
+## Features
+
+1. Users can connect wallet from metamask or with wallet connect
+2. Seller can list an item, which mints an 1155 NFT
```
Comment (atkinsonholly) [inline src/docs/reference-application/introduction.md:27]: which mints an ERC1155.

File: src/docs/reference-application/introduction.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+3. Seller can view her listings and void any remaining quantities that haven't
+   been committed to
+4. Seller can track the lifecycle of her listings of minted 721 NFTs committed
```
Comment (atkinsonholly) [inline src/docs/reference-application/introduction.md:30]: minted ERC721 tokens

File: src/docs/reference-application.md (hunk 1 above)
Comment (hswopeams) [reply src/docs/reference-application.md:8]: Will be changed in next commit.

File: src/docs/reference-application/development-setup.md (hunk 2 above)
Comment (hswopeams) [reply src/docs/reference-application/development-setup.md:18]: Will be changed in next commit.

File: src/docs/reference-application/introduction.md (hunk 3 above)
Comment (hswopeams) [reply src/docs/reference-application/introduction.md:16]: Will be changed in next commit.

File: src/docs/reference-application/introduction.md (hunk 4 above)
Comment (hswopeams) [reply src/docs/reference-application/introduction.md:27]: Will be changed in next commit.

File: src/docs/reference-application/introduction.md (hunk 5 above)
Comment (hswopeams) [reply src/docs/reference-application/introduction.md:30]: Will be changed in next commit.

File: README.md [hunk 6]
```diff
@@ -15,6 +15,16 @@ To start up local development servers:
     ./go content:jekyll:serve
     ```
 * Navigate to [http://localhost:4000](http://localhost:4000)
+* If you develop in a guest VM but use the browser on you host machine, you may not be able to access the local site using `localhost`. In this case, add the following to `bsn-local-default.yaml` to start the Jekyll server on 0.0.0.0:
+```
+    # deployment
+    host: 0.0.0.0
+
+```
+   ** DO NOT CHECK THIS CHANGE IN **
```
Comment (atkinsonholly) [inline README.md:24]: @hswopeams is this protected?

**Review by atkinsonholly — APPROVED** [review]

**Pull request merged**